R/count_teleconnections.R

```diff
@@ -321,6 +321,11 @@ count_watershed_teleconnections <- function(data_dir,
             .[["Reclass"]] %>% unique()
         }) %>% unlist() %>% unique() %>% length() -> n_economic_sectors
 
+        # waste discharge per person
+        map(city_watershed_data, function(x){
+          x$metrics %>% filter(metric == "waste discharge per person") %>%
+            .[["value"]]
+        }) %>% unlist() %>% sum() -> wasteMGD_per_person
```

Comment:
Please check this sum. Are you summing discharge per person across watersheds?

Reply:
Appropriate method here would be to sum the discharge across watersheds, sum the associated population across watersheds and then compute the per person average here.

Reply:
You are absolutely right on that. I'll make sure to do the calculations after those are summed up!